src/clue/test/constants.py

```diff
@@ -0,0 +1,2 @@
+IMG_DIR_NAME = "img"
+SCREEN_HEIGHT_WIDTH = 240
```

Comment:
In places, where we used this constants file, can we use the one that the clue uses?